packages/twenty-front/src/modules/action-menu/states/contexts/ActionMenuContext.ts

```diff
@@ -0,0 +1,11 @@
+import { createContext } from 'react';
+
+export type ActionMenuContextType = {
```

Comment:
do we need to export it? usually if we export we want a separate file (here in types)